lib/services/identity.go

```diff
@@ -269,12 +269,6 @@ type Identity interface {
 	// CreateUserRecoveryAttempt logs user recovery attempt.
 	CreateUserRecoveryAttempt(ctx context.Context, user string, attempt *types.RecoveryAttempt) error
 
-	// GetUserRecoveryAttempts returns user recovery attempts sorted by oldest to latest time.
-	GetUserRecoveryAttempts(ctx context.Context, user string) ([]*types.RecoveryAttempt, error)
```

Comment:
Can we get rid of `GetUserRecoveryAttempts` completely? I see you removed it from the interface, but the implementation remains, so we have an orphaned function that nothing is calling left behind.

Reply:
This was an oversight, removed, thank you!

Reply:
I still see some references to `GetUserRecoveryAttempts`, `DeleteUserRecoveryAttempts`, and `CreateUserRecoveryAttempt` in the codebase. These can be removed, along with the `recoveryAttemptsPrefix` constant.

Reply:
I removed `recoveryAttemptsPrefix`, that was missed.  But can you point me to the other cases?  I just did a grep for the three CRUD functions and can't find any reference to any of them.  Thank you!